adhocracy4/ckeditor/storage.py

```diff
@@ -0,0 +1,12 @@
+import os
+from urllib.parse import urljoin
+
+from django.conf import settings
+from django.core.files.storage import FileSystemStorage
+
+
+class CustomStorage(FileSystemStorage):
+    """Custom storage to store uploads in a subfolder called uploads"""
+
+    location = os.path.join(settings.MEDIA_ROOT, "uploads")
```

Comment:
I'd specify those are from the ck editor uploads, so maybe a prefix or a suffix to `uploads`

Reply:
This breaks existing behavior though, the old ckeditor uploads them to `uploads`. I'd prefer to keep the current behavior for now. We have another story which will touch the upload behavior (file names, path), so we can change it as part of that it that's ok?